rackcity/utils/rackcity_utils.py

```diff
@@ -174,53 +173,37 @@ def validate_location_modification(data, existing_asset, user, change_plan=None)
     asset_rack_position = existing_asset.rack_position
     asset_chassis_slot = existing_asset.chassis_slot
     asset_height = existing_asset.model.height
-    if "rack_position" in data:
+    if "rack_position" in data and data["rack_position"]:
         try:
             asset_rack_position = int(data["rack_position"])
         except ValueError:
             raise Exception("Field 'rack_position' must be of type int.")
 
-    if "chassis_slot" in data:
+    if "chassis_slot" in data and data["chassis_slot"]:
```

Comment:
these values are sent as null from the front end now so casting to `int` was causing uncaught error. also BTW for everyone ValueError catches non-int problems but it doesn't catch None type objects (that throws a TypeError)

Reply:
Nice catches